### PR TITLE
Add Vocaloid waifu

### PR DESCRIPTION
**Pull request by burstfour**

### CODE REVIEW

File: waifus/hatsune miku.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	"type" : 0,
+	"english-name" : "Hatsune Miku",
+  "alias" : ["Miku","Hatsune","Vocaloid],
+	"image" : "https://ec.crypton.co.jp/img/special/vocaloid/img_MIKU_us.png",
+	"japanese-name" : "初音ミク",
+  "year" : 2007,
+	"month" : 8,
+	"day-of-month" : 31,
+	"age-in-show": 16,
+	"sources" : ["https://ec.crypton.co.jp/pages/prod/vocaloid/cv01_us"]
```
Comment (yourWaifu) [inline waifus/hatsune miku.json]: this source doesn't have any age info, but I found it on https://en.wikipedia.org/wiki/Hatsune_Miku

File: megurine luka.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Megurine Luka",
+	"image" : "https://vignette.wikia.nocookie.net/vocaloid/images/3/39/Luka_v4x_final.png",
+	"japanese-name" : "巡音ルカ",
+  "year" : 2009,
```
Comment (yourWaifu) [inline megurine luka.json]: indent please

**Comment by burstfour** [comment]
Thanks for the correction!
I have committed the fix just now.

**Comment by burstfour** [comment]
I added my last Vocaloid waifu just now.
You can review the change anytime!

**Pull request merged**